app/src/main/java/com/example/stretchy/features/executetraining/ui/composable/ExecuteTrainingComposable.kt

```diff
@@ -58,14 +66,22 @@ fun ExecuteTrainingComposable(
                                 ExerciseComposable(
                                     exerciseName = item.name,
                                     nextExerciseName = item.nextExercise,
-                                    currentTime = item.currentTime,
-                                    totalTime = item.totalTime
+                                    currentExerciseTime = item.currentTime,
+                                    totalExerciseTime = item.totalExerciseTime,
+                                    trainingProgressPercent = item.trainingProgressPercent,
+                                    disableExerciseAnimation = disableExerciseAnimation,
+                                    initialProgressBarValue = initialProgressBarValue,
+                                    onExerciseEnd = { initialProgressBarValue = it }
```

Comment:
this callback makes 0 sense, remove it and use by remember if you want to keep some value in composable function